cores/arduino/main.cpp

```diff
@@ -45,6 +45,9 @@ int main( void )
 
 	for (;;)
 	{
+		// Define by Ethernet library. It is defined as __weak.
+		stm32_eth_scheduler();
```

Comment:
This function is defined as weak in ethernet.c, when the library NativeEthernet is not included, this function is empty. 
And to call it in the loop, we must create a function that the user should call (this function will not be refered in the Arduino documentation). 
I thought it was easier to leave that in background.